### DISCUSSION TITLE
Safe Exam Browser crashes on iPad

### DISCUSSION THREAD

**Discussion by LucaKrenger** [SafeExamBrowser/seb-mac#191] [Q&A] [answered]
Hello

We tried to set up an exam on iOS. The Configuration File used for the exam works on Windows but led the App to crash on two independent iPads. (iPad Pro 9.7", IOS 15.3.1 and iPad Pro (3. Generation), ios-Version 15.3.1)

Do you have any suggestions why it crashed?

Thank you for your response.

**Comment by danschlet** [comment, marked as answer]
There is one particular bug in the current SEB for iOS version 3.0.1, where after starting SEB with a seb(s):// link, it would crash. One workaround is to first start SEB with its icon on the home screen and then go to the regular browser to start the exam with the seb(s) link. Or to configure the (permanent) client settings in SEB on the device to for example load the start page of your exam system/LMS or a portal/landing page with seb(s) links to the exams, when SEB is started with its icon.

This bug is fixed in SEB 3.1, the public beta is available in TestFlight (if not yet registered for the TestFlight betas, you can use this link on an iPad/iPhone to get the beta: https://testflight.apple.com/join/egxLmc4j). We plan to release the final version in the next couple weeks, so testing the almost final beta is highly recommended and any feedback helpful for us.
